tests/resources/aria-at-test-run.mjs

```diff
@@ -261,9 +243,9 @@ export function instructionDocument(resultState, hooks) {
         change: atOutput => hooks.setCommandOutput({ commandIndex, atOutput }),
       },
       assertionsHeader: {
-        descriptionHeader: 'Assertion',
-        passHeader: 'Success case',
-        failHeader: 'Failure cases',
+        descriptionHeader: `${
+          resultState.assertionResponseQuestion ?? 'Which statements are true about the response to'
```

Comment:
I'm guessing this nullish check was for it missing, when building against v1. But my previous comment should address not needing it, if you'd like to remove it here. It's not unlike this section to have default hardcoded fallbacks so I don't feel strongly on removing it or not.

Reply:
Removed. Thanks!